snapcraft/integrations/travis.py

```diff
@@ -0,0 +1,164 @@
+# -*- Mode:Python; indent-tabs-mode:nil; tab-width:4 -*-
+#
+# Copyright (C) 2015 Canonical Ltd
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Snapcraft integration for Travis (CI).
+
+This command currently depends on working `travis` CLI environment and
+a previously initialised Travis project (`.travis.yml`).
+
+Make sure your Travis project is also configured to "Build pushes", this
+way every new push to `master` will result in a new snap revision on the
+Store.
```

Comment:
the travis commands added are configured to run only on changes to master (` if [ "$TRAVIS_PULL_REQUEST" = "false" ];`)